src/CUTEst.jl

```diff
@@ -28,12 +28,14 @@ const cutest_problems_path = joinpath(dirname(@__FILE__), "..", "deps", "files")
 isdir(cutest_problems_path) || mkpath(cutest_problems_path)
 
 function __init__()
```

Comment:
```suggestion
function __init__()
```
Could you just add a comment around to say why we don't do anything for Windows here?

Reply:
It's because we don't have `bash` on Windows. I will add a comment in a different PR.